### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by qmnl** [tj/better-assert#21]
The current version is 8 commits behind. Furthermore, the newest version of the npm package does not yet include the license (see #11).

**Comment by EdwinOtten** [comment]
I agree. I was using the [license-compatibility-checker](https://www.npmjs.com/package/license-compatibility-checker) today and it marks this package as possibly incompatible because the latest release doesn't include #11 yet.

@tj I realise you are probably very busy and requests like this are not very urgent. But if you are able to do it, I am sure you'd make a lot of people happy 😃

**Comment by EdwinOtten** [comment]
The same applies to callsite ([PR 13](https://github.com/tj/callsite/issues/13)).

If it would help, I can setup automated deployment for you (using GitHub Actions to trigger a deployment whenever a new commit to master is pushed). Let me know if can help you with that.
